test/HttpServerFactoryTest.php

```diff
@@ -294,4 +296,38 @@ public function testServerCanBeStartedForKnownSocketTypeCombinations($socketType
         Process::wait(true);
         $this->assertSame('Server Started', $output);
     }
+
+    public function testFactoryCanBeEnableCoroutine()
+    {
+        $this->container->get('config')->willReturn([
+            'zend-expressive-swoole' => [
+                'enable_coroutine' => true,
+            ],
+        ]);
+        $factory = new HttpServerFactory();
+        $factory($this->container->reveal());
+
+        if (! method_exists(SwooleRuntime::class, 'enableCoroutine')) {
+            return ;
```

Comment:
I believe this is due to Swoole prev to v4, right?
We shouldn't just return, but test should be marked as skipped in that case.
I believe it should be at the beginning of the test.

Reply:
I agree with @webimpress here. Let's get that in place, and I'll do final review.

Thanks, @Moln

Reply:
I've pushed this change to your branch.

Reply:
👍 👍 👍 

Sorry, I'm too busy at work. And thanks for your time to review and change this branch.